pkg/ui/src/views/cluster/containers/nodesOverview/index.tsx

```diff
@@ -112,22 +114,47 @@ class LiveNodeList extends React.Component<NodeCategoryListProps, {}> {
               },
               sort: (ns) => ns.started_at,
             },
-            // Used Capacity - displays the total persisted bytes maintained by the node.
-            {
-              title: "Used Capacity",
-              cell: (ns) => Bytes(BytesUsed(ns)),
-              sort: (ns) => BytesUsed(ns),
-            },
             // Replicas - displays the total number of replicas on the node.
             {
               title: "Replicas",
               cell: (ns) => ns.metrics[MetricConstants.replicas].toString(),
               sort: (ns) => ns.metrics[MetricConstants.replicas],
             },
+            // CPUs - the number of CPUs on this node
+            {
+              title: "CPUs",
+              cell: (ns) => ns.num_cpus,
+            },
+            // Used Capacity - displays the total persisted bytes maintained by the node.
+            {
+              title: "Capacity Usage",
+              cell: (ns) => {
+                const { usable } = nodeCapacityStats(ns);
+                const used = BytesUsed(ns);
+                return (
+                  <span title={`Total: ${BytesWithPrecision(usable, 0)}`}>
+                    {BytesWithPrecision(used, 0)}
+                    {" "}
+                    ({Math.round(used / usable * 100)}%)
+                  </span>
+                );
+              },
+              sort: (ns) => BytesUsed(ns),
+            },
             // Mem Usage - total memory being used on this node.
             {
               title: "Mem Usage",
-              cell: (ns) => Bytes(ns.metrics[MetricConstants.rss]),
+              cell: (ns) => {
+                const used = ns.metrics[MetricConstants.rss];
+                const available = FixLong(ns.total_system_memory).toNumber();
```

Comment:
i'd prefer if we did similar treatment in `nodeSumStats` for memory as we do for capacity, to keep things consistent